docs_app/src/styles/2-modules/_migration-timeline-pages.scss

```diff
@@ -0,0 +1,107 @@
+.migration-timeline-navigation {
+  .mat-chip-list:focus  {
+    outline: 0;
```

Comment:
AFAIK outlines are important for accessibility

Reply:
done